lib/task-graph.js

```diff
@@ -59,13 +59,57 @@ function taskGraphFactory(
         };
         if (this.context.target) {
             this.logContext.id = this.context.target;
-        }
+       }
```

Comment:
The previous indent was correct, but you changed it to wrong indent, please revert this change.